module2-database-technologies/r2.2-sql-queries/README.md

```diff
@@ -0,0 +1,69 @@
+# SQL queries
+
+Now that we understand the general concept behind relational data, how do we
+query it? Note that every SQL database may have slightly different syntax in
+terms of quotations, capitalization, commas, etc., so be sure to check. In these
+examples, we will be using MySQL.
+
+Suppose you have an existing MySQL database. To create the `customer` table (we
+will now lowercase the table names by convention), you would use the `CREATE
+TABLE` statement. Note that every column has a data type and can also be given additional definition rules.
+
+[`AUTO_INCREMENT`](https://www.guru99.com/auto-increment.html) is used to
+automatically generated the IDs; when inserting a row, you do not need to
+provide the ID, it is automatically generated by incrementing the previous ID. We will see this in our example queries later. `NOT NULL` means that the user cannot provide a `NULL` value for values in this column. `PRIMARY KEY` is used to denote a column as the primary key field, and it is often used in conjunction with `AUTO INCREMENT` for databases with numeric IDs.
+
+There are many options when creating columns. Refer to the [`CREATE TABLE`
+documentation](https://dev.mysql.com/doc/refman/8.0/en/create-table.html).
```

Comment:
Let's not use both back ticks and link formatting. If CREATE TABLE needs to be in back ticks, then we can just add the link on "documentation".